Add SQL: Dates term entries (#839)
Codecademy/docs: content/sql/concepts/dates/terms/dateadd/dateadd.md

---
Title: 'DATEADD()'

> Description: 'Adds a time or date interval to a specified date and returns it. Versions are available in SQL Server and MySQL.'

Reviewer: ```suggestion
Description: 'Returns a date/time interval added to a specified date. Versions are available in SQL Server and MySQL.'
```
